maptile/tile.go

```diff
@@ -226,6 +226,31 @@ func (t Tile) Children() Tiles {
 	}
 }
 
+// ChildrenInZoomRange returns all the children tiles of tile from ranges [zoomStart, zoomEnd], both ends inclusive.
+func ChildrenInZoomRange(tile Tile, zoomStart, zoomEnd Zoom) Tiles {
+	if zoomStart < tile.Z || zoomEnd < tile.Z {
+		panic("tile.Z is greater than zoomStart or zoomEnd")
+	}
+
+	zDeltaStart := zoomStart - tile.Z
+	zDeltaEnd := zoomEnd - tile.Z
+
+	res := make([]Tile, 0)
+
+	for d := zDeltaStart; d <= zDeltaEnd; d++ {
+		xTopLeft := tile.X << d
```

Comment:
this variable seems more like `left` or `xStart`

Then the next one is more like `top` or `yStart`

Reply:
Fixed in next commit.